<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>contactslist</class>
 <widget class="QWidget" name="contactslist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="friendsList">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>54</y>
     <width>200</width>
     <height>646</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="autoScrollMargin">
    <number>16</number>
   </property>
  </widget>
  <widget class="QPushButton" name="addFriendButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>100</width>
     <height>54</height>
    </rect>
   </property>
   <property name="text">
    <string>添加好友</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkNotificationButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>100</width>
     <height>54</height>
    </rect>
   </property>
   <property name="text">
    <string>查看通知</string>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(237, 255, 254);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="notificationList">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>54</y>
     <width>200</width>
     <height>646</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="autoScrollMargin">
    <number>16</number>
   </property>
  </widget>
  <widget class="QLabel" name="noticeNewLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700; color:#ff5500;&quot;&gt;New&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>friendsList</zorder>
  <zorder>addFriendButton</zorder>
  <zorder>checkNotificationButton</zorder>
  <zorder>notificationList</zorder>
  <zorder>noticeNewLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
